src/v/pandaproxy/json/iobuf.h

```diff
@@ -84,6 +86,11 @@ class rjson_serialize_impl<iobuf> {
         return w.String(iobuf_to_base64(buf));
     };
 
+    bool encode_json(rapidjson::Writer<rapidjson::StringBuffer>& w, iobuf buf) {
+        iobuf_parser p(std::move(buf));
+        return w.String(p.read_string(p.bytes_left()));
```

Comment:
This is the bug - it shouldn't be quoted as a string.

Reply:
`p.read_bytes()` ?